base/Command.js

```diff
@@ -84,5 +67,10 @@ class Command {
   async run(message, args, level) { // eslint-disable-line no-unused-vars
     throw new Error(`Command ${this.constructor.name} doesn't provide a run method.`); 
   }
+
+  async init() {
+    // ¯\_(ツ)_/¯
```

Comment:
This comment could've been more informative

Reply:
Its there for commandLoad nothing else